_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1732,35 +1743,22 @@ Set the variables for the `account-baseline-security` module in this environment
 [source,hcl]
 ----
 locals {
-  aws_region = "us-east-1"
-
-  accounts = {
-    logs     = "409800740445"
-    security = "673123100581"
-    shared   = "384759303421"
-    dev      = "293847503945"
-    stage    = "384924092834"
-    prod     = "784260063686"
-    root     = "216044045972"
-  }
+  # A local for more convenient access to the accounts map.
+  accounts = local.common_vars.locals.accounts
 
   # Both buckets are created in the logs account by account-baseline-root
-  config_s3_bucket_name     = "acme-config-bucket-logs"
-  cloudtrail_s3_bucket_name = "acme-cloudtrail-logs"
+  config_s3_bucket_name     = local.common_vars.locals.config_s3_bucket_name
+  cloudtrail_s3_bucket_name = local.common_vars.locals.cloudtrail_s3_bucket_name
 
   # The Cloudtrail KMS Key is deployed at the logs account but it's value is an output from the root account.
-  cloudtrail_kms_key_arn = "arn:aws:kms:us-east-1:216044045972:alias/cloudtrail-acme"
+  cloudtrail_kms_key_arn = local.common_vars.locals.cloudtrail_kms_key_arn
 
   # A local for convenient access to the security account root ARN.
   security_account_root_arn = "arn:aws:iam::${local.accounts.security}:root"
 }
 input = {
-  # Fill in the region you want to use (only used for API calls) and the IDs of your AWS accounts (see the locals above)
-  aws_region     = local.aws_region
```

Comment:
Where is `aws_region` being defined?

Reply:
Hm not sure what was envisioned here, but there's also `region.hcl` file that we can define the same way as the `account.hcl`. I actually don't think it's necessary at this point - so am tempted to just leave it here. Thoughts?